docs/support.md

```diff
@@ -6,11 +6,11 @@ You can access our pro support if you subscribe to any of these plans:
 * Enterprise
 * Premium
 
-The better the plan, the faster the support will be with higher priority.
+Higher tier support plans include faster ticket response times. Paid support plans and response times are based on the plan you have, plans can be [reviewed here](https://xen-orchestra.com/#!/xo-pricing).
 
 ## Free limited support
 
-In the free edition of Xen Orchestra, you also have ability to open support ticket. We will do our best to assist you, however, this support is limited and has no guaranteed response time. 
+With the free edition of Xen Orchestra, you can open support tickets and we will do our best to assist you, however, this support is limited and is not guaranteed in regards to response times or resolutions offered.
```

Comment:
Maybe we should make a clear distinction between XOA and "the sources". So in the previous paragraph, explaining that pro support comes with XOA as a turnkey/validated platform, and then explain that even XOA Free got pro support on the same platform but without response time guarantee. Then the next title/paragraph is a good transition toward "the sources" and it seems logical. What do you think?